responsibleai/responsibleai/_managers/causal_manager.py

```diff
@@ -92,6 +92,13 @@ def add(
         :param heterogeneity_features: Features that mediate the causal effect.
         :type heterogeneity_features: list
         :param nuisance_model: Model type to use for nuisance estimation.
```

Comment:
There's some redundancy there between l.94 and 96. Perhaps the following is better:

```
:param nuisance_model: Model type to use for nuisance estimation.
            (i.e. for estimating propensity models or models of how controls predict the outcome)
            It is one of {'linear', 'automl'}, optional (default='linear').
            If 'linear', then LassoCV (for regression) and
            LogisticRegressionCV (for classification) are used.
            If 'automl', then a k-fold cross-validation and model selection
            is performed among several models and the best is chosen.
```

Reply:
The line 'estimating propensity models or models of how controls predict the outcome' still isn't terribly clear to me.... I don't understand what this is really doing.

Reply:
I took this doc from econml. For now I have removed this bit and only added description for linear and automl models.

Reply:
@riedgar-ms, I have updated the description for nuisance_model and hetrogenity_model. Can you take a look and approve?